### PR TITLE
Adding compression quality as parameter of ResizeOption. Fixes #41.

### LINKED ISSUE #41: Jpeg compression quatlity as parameter for rememberImagePickerLauncher
It would be great if together with resize options we can also pass jpeg compression rate to the rememberImagePickerLauncher. 
As far as i see currently it's always set to 1.0 in UIImage.toByteArray() function called for picked images before returning on iOS side. And should be as easy to change on android side in resizeImage function of PeekabooImageResizer. 

Thanks for the library!

### PR DESCRIPTION
**Pull request by shtolik**
Adding compression quality as parameter of ResizeOption. Fixes #41

### CODE REVIEW

File: peekaboo-image-picker/src/iosMain/kotlin/com/preat/peekaboo/image/picker/ImagePickerLauncher.ios.kt [hunk 1]
```diff
@@ -121,8 +121,8 @@ actual fun rememberImagePickerLauncher(
 }
 
 @OptIn(ExperimentalForeignApi::class)
-private fun UIImage.toByteArray(): ByteArray {
-    val jpegData = UIImageJPEGRepresentation(this, 1.0)!!
+private fun UIImage.toByteArray(compressionQuality: Double): ByteArray {
+    val jpegData = UIImageJPEGRepresentation(this, compressionQuality)!!
```
Comment (onseok) [inline peekaboo-image-picker/src/iosMain/kotlin/com/preat/peekaboo/image/picker/ImagePickerLauncher.ios.kt:125]: As I mentioned in the comment, let's change this point using a validation check for the compression quality. 
```kotlin
private fun UIImage.toByteArray(compressionQuality: Double): ByteArray {
    val validCompressionQuality = compressionQuality.coerceIn(0.0, 1.0)
    val jpegData = UIImageJPEGRepresentation(this, validCompressionQuality)!!
    // ...
}
```

File: peekaboo-image-picker/src/androidMain/kotlin/com/preat/peekaboo/image/picker/PeekabooImageResizer.kt [hunk 2]
```diff
@@ -90,7 +92,7 @@ internal object PeekabooImageResizer {
             val filteredBitmap = applyFilter(rotatedBitmap, filterOptions)
 
             ByteArrayOutputStream().use { byteArrayOutputStream ->
-                filteredBitmap.compress(Bitmap.CompressFormat.JPEG, 100, byteArrayOutputStream)
+                filteredBitmap.compress(Bitmap.CompressFormat.JPEG, (100 * compression).toInt(), byteArrayOutputStream)
```
Comment (onseok) [inline peekaboo-image-picker/src/androidMain/kotlin/com/preat/peekaboo/image/picker/PeekabooImageResizer.kt:95]: We can change this like the below.
```kotlin
val validatedCompression = compression.coerceIn(0.0, 1.0)
filteredBitmap.compress(Bitmap.CompressFormat.JPEG, (100 * validatedCompression).toInt(), byteArrayOutputStream)
```

**Review by onseok — CHANGES_REQUESTED** [review]
I've left a brief review. I'd appreciate it if you could check it and make any necessary changes. Thank you, @shtolik

File: peekaboo-image-picker/src/iosMain/kotlin/com/preat/peekaboo/image/picker/ImagePickerLauncher.ios.kt (hunk 1 above)
Comment (l2hyunwoo) [reply peekaboo-image-picker/src/iosMain/kotlin/com/preat/peekaboo/image/picker/ImagePickerLauncher.ios.kt:125]: I think attaching [FloatRange](https://developer.android.com/reference/kotlin/androidx/annotation/FloatRange) annotation should be good choice for readability

Comment (shtolik) [reply peekaboo-image-picker/src/iosMain/kotlin/com/preat/peekaboo/image/picker/ImagePickerLauncher.ios.kt]: Added FloatRange annotation (and annotation lib to toml and project)

**Comment by shtolik** [comment]
Pushed changes fixing suggestions. 
Sorry for late fix

File: gradle/libs.versions.toml [hunk 3]
```diff
@@ -49,6 +49,7 @@ androidx-exifinterface = { group = "androidx.exifinterface", name = "exifinterfa
 androidx-lifecycle-viewmodel-compose = { group = "androidx.lifecycle", name = "lifecycle-viewmodel-compose", version.ref = "androidx-lifecycleViewmodelCompose" }
 paging-common = { module = "app.cash.paging:paging-common", version.ref = "multiplatform-paging" }
 paging-compose-common = { module = "app.cash.paging:paging-compose-common", version.ref = "multiplatform-paging" }
+annotation = { module = "androidx.annotation:annotation", version = "1.7.1" }
```
Comment (l2hyunwoo) [inline gradle/libs.versions.toml:52]: Can you write specific version reference like above case?

Comment (shtolik) [reply gradle/libs.versions.toml]: I usually make a version variable if there are several places it's used, but yeah, sure, fixed.

**Pull request merged**